src/aplContainer/commands/changeViewportProfile.ts

```diff
@@ -32,12 +32,12 @@ export class ChangeViewportProfileCommand extends AbstractCommand<void> {
         try {
             const aplPreviewPanel: vscode.WebviewPanel = this.aplPreviewWebView.getPanel();
             // change viewport profile before preview, no panel 
-            if (!aplPreviewPanel || !aplPreviewPanel.visible) {
-                throw loggableAskError(ERROR_MESSAGES.CHANGE_VIEWPORT_PROFILE_NO_APL_PREVIEW, undefined, true);
+            if (aplPreviewPanel === undefined || !aplPreviewPanel.visible) {
+                throw logAskError(ERROR_MESSAGES.CHANGE_VIEWPORT_PROFILE_NO_APL_PREVIEW, undefined, true);
```

Comment:
why change `loggableAskError` to `logAskError` in this PR ?

Reply:
It was causing spellCheck errors, so refactored it to `logAskError` any issue on changing ?